src/role_sourcer.js

```diff
@@ -183,7 +183,14 @@ function getContainerConstructionSite(creep) {
  */
 function maintainContainer(creep) {
   if (creep.inBase()) {
-    return false;
+    const links = creep.pos.findInRangePropertyFilter(FIND_STRUCTURES, 1, 'structureType', [STRUCTURE_LINK]);
+    if (links.length) {
+      const containerConstructionSite = getContainerConstructionSite(creep);
+      if (containerConstructionSite) {
+        containerConstructionSite.remove();
+      }
+      return false;
+    }
```

Comment:
```suggestion
    return creep.room.controller.level < 6;
```

How about this?
Doing find operations per tick per sourcer costs CPU.
At RCL 6 we should have all links at the sourcer.
If we don't maintain the container they will decay at some point in time, so no need to touch.
What do you think?

Reply:
sounds good!